frontends/web/src/components/devices/bitbox02bootloader/bitbox02bootloader.tsx

```diff
@@ -134,7 +138,7 @@ class BitBox02Bootloader extends Component<Props, State> {
                         <Button
                             primary
                             onClick={this.upgradeFirmware}>
-                            {t('bootloader.button')}
+                            {t(erased ? 'bootloader.buttonInstall' : 'bootloader.button')}
```

Comment:
What does `devices/bitbox02-bootloader/${deviceID}/erased` return?

I assume `erased` would return true when the device has been reset, or does `erased` mean it is a fresh device with no firmware?

Bit confused.

Reply:
yeah erased simply means there's no firmware on it. it is also how the device is shipped to customers. I added a comment.